### PR TITLE
Updated BUILD instructions.

### PR DESCRIPTION
**Pull request by jluhrs**

### CODE REVIEW

File: BUILD [hunk 1]
```diff
@@ -127,19 +127,41 @@ using the command
...
+  1. Update all the SNAPSHOT version to final versions.
+  2. Commit the changes to the versions.
+  3. Tag the repository with the new version for GMP (GMP-R<version>)
+  4. Increment all the version numbers and add the SNAPSHOT prefix to them, in preparation for the next
+     development cycle.
+  5. Commit the changes to the versions.
+  6. Push all the changes and the new tag to the gemini-hlsw repository in Github.
+
+Once that is ready you can actually perform the release with the command (you must use the location of your
+local maven-repo working copy):
```
Comment (cquiroz) [inline BUILD:158]: Maybe explicit that this refers to the `maven-repo` on github under `geminihlsw`

**Review by cquiroz — APPROVED** [review]
Looks good it only needs to run `%s/Maven/maven/g`

File: BUILD (hunk 1 above)
Comment (jluhrs) [reply BUILD:158]: That is already mentioned in the third prerequisite.

Comment (cquiroz) [reply BUILD:158]: Ok, I missed that

**Pull request merged**